client/src/components/Allocations/AllocationsModals/AllocationsTeamViewModal/AllocationsTeamViewModal.jsx

```diff
@@ -1,30 +1,23 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import { number, bool, func } from 'prop-types';
-import {
-  Modal,
-  ModalHeader,
-  ModalBody,
-  Container,
-  Col,
-  Row,
-  Button,
-} from 'reactstrap';
-import { useSelector } from 'react-redux';
+import { Modal, ModalHeader, ModalBody, Container, Col, Row } from 'reactstrap';
+import { Tab, Tabs } from '@material-ui/core';
```

Comment:
@wesleyboar Thoughts on using more from material? Looks good, but does have a click animation.
![Screen Shot 2022-05-25 at 5 11 36 PM](https://user-images.githubusercontent.com/20326896/170375859-e5ede3c3-9a3c-4506-9b47-4eae1e240b22.png)

Reply:
PR approved and merged.